text/0732-cdk-cli-telemetry.md

```diff
@@ -115,6 +98,27 @@ Alternatively, you can run a new CDK CLI command:
 
 This will record the action to your local `cdk.json` file, which affects the current CDK App only.
 
+We'll also add a global CLI option you can append to any CDK CLI command: `--disable-telemetry`.
+If passed in, we will not collect library metadata or CLI telemetry for that command.
+The old `--no-version-reporting` flag is no longer recommended, but will also disable both library
+metadata and CLI telemetry.
```

Comment:
Add a disclaimer that if the `analyticsReporting` property is set on the stack level, it takes precedence over any CLI flag.